docs/DeveloperGuide.md

```diff
@@ -464,39 +479,39 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 3. HelloFile open the parent folder with the tag.
 
     Use case ends.
-    
+        
 **Extension**
 
 * 2a. HelloFile detects that the tag is invalid.
 
 	* 2a1. HelloFile prompts the user that the tag is wrong.
-	
+		
         Use case resumes from step 1.
-        
+                
 * 3a. HelloFile detects that the target file does not exist.
 	
 	* 3a1. HelloFile prompts that the target file does not exist.
-		
+			
 	    Use case resumes from step 1.
-		
+	    		
 * 3b. HelloFile detects no default application to open the file.
 	
 	* 3b1. HelloFile prompts the user to choose an application to open the file.
-		
+			
         Use case resumes form step 1.
-		
+        		
 * 3c. HelloFile encounters an exception when opening the file.
 	
 	* 3c1. HelloFile shows the error message.
-		
+			
 		Use case resumes from step 1.
-		
+				
 * *a. HelloFile has no permission to read or write to the file system.
 
 	* *a1. HelloFile prompts the user that it needs the read and write permission.
 	
-	    Use case resumes.
-
+		    Use case resumes.
+		    
```

Comment:
I think you broke some of my parts. You can review this markdown file again to find out more.

Before
![image](https://user-images.githubusercontent.com/51979232/94297572-4f12d400-ff97-11ea-8419-27caf77e84c4.png)

After
![image](https://user-images.githubusercontent.com/51979232/94297495-2be82480-ff97-11ea-997e-6c8283de59ef.png)

Reply:
I have night mode enabled so it is more obvious.

Reply:
Okay, I have restore to original, sorry for the different input. I merge code to master and there were conflicts.

Reply:
There's still something wrong here.

![image](https://user-images.githubusercontent.com/51979232/94329672-c3796180-ffef-11ea-97ea-77bebb01e5df.png)